### PR TITLE
Updates and improvements

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR improve the markdown style, updates the section on clang-format and that one on updating MEI3 to MEI4.

### CODE REVIEW

File: _book/06-installing-or-building-from-sources/03-python.md [hunk 1]
```diff
@@ -116,8 +116,6 @@ If you want to enable or disable other specific options, you can do:
 cmake ../cmake -B python -DBUILD_AS_PYTHON=ON -DNO_PAE_SUPPORT=ON
 ```
 
-*Installation with CMake has not be tested yet*
-
```
Comment (lpugin) [inline _book/06-installing-or-building-from-sources/03-python.md:120]: You've tested it?

Comment (rettinghaus) [reply _book/06-installing-or-building-from-sources/03-python.md:120]: Yes, tested on macOS and Ubuntu. That should be sufficient, I guess.

Comment (lpugin) [reply _book/06-installing-or-building-from-sources/03-python.md:120]: Great. Where is the python module installed? You ran `sudo make install`?

Comment (rettinghaus) [reply _book/06-installing-or-building-from-sources/03-python.md:120]: No, I built it in place and added the path to PYTHONOPATH. I can try to install it.

Comment (lpugin) [reply _book/06-installing-or-building-from-sources/03-python.md:120]: OK, then some more information about that would be useful

Comment (rettinghaus) [reply _book/06-installing-or-building-from-sources/03-python.md]: I've put it back in for now.

File: _book/05-toolkit-reference/01-input-formats.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ The native input format for Verovio is MEI. Verovio supports MEI as input format
 
 When loading MEI data into Verovio and outputting MEI, elements that are not supported by Verovio will be ignored. This means that they are not loaded into memory and will not be preserved in the MEI output. This includes the element themselves, but also any descendant they might have. A warning will be given in the console. For example:
 
-```bash
+```console
```
Comment (lpugin) [inline _book/05-toolkit-reference/01-input-formats.md:261]: console has no CSS label. Maybe we can just remove `bash`?

Comment (rettinghaus) [reply _book/05-toolkit-reference/01-input-formats.md]: Not ideal, but ok for now.

**Pull request merged**